### PR TITLE
[bugfix/PLAYER-5272]  loading spinner is displayed on IMA ads

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Hardcore value for "buffer" was changed from 0 to 1 to prevent showing a spinner on Ad screen.

### CODE REVIEW

File: js/google_ima.js [hunk 1]
```diff
@@ -2894,12 +2894,25 @@ require("../html5-common/js/utils/utils.js");
...
+     * )
+     * set it to 1 (use 1 instead of 0 to avoid the problem of displaying Spinner on the AdScreen)
+     * @param {String} eventname - event name
+     * @param {Number} currentTime - current time
+     * @param {Number} duration - a video duration
+     * @param {Number} buffer - value of buffer
+     * @link https://groups.google.com/forum/#!topic/ima-sdk/zRNKpKNSukM
+     */
+    const raisePlayhead = _.bind((eventname, currentTime, duration, buffer=1) => {
+      console.log('BBB buffer', buffer);
```
Comment (zelibobla) [inline js/google_ima.js]: Forgotten?

File: js/google_ima.js [hunk 2]
```diff
@@ -2894,12 +2894,25 @@ require("../html5-common/js/utils/utils.js");
...
       notifyIfInControl(eventname,
         { "currentTime" : currentTime,
           "duration" : duration,
-          "buffer" : 0,
+          "buffer" : buffer,
           "seekRange" : { "begin" : 0, "end" : 0 } });
     }, this);
```
Comment (zelibobla) [inline js/google_ima.js]:
```suggestion
  const seekRange = { begin: 0, end: 0};
  notifyIfInControl(eventName, { currentTime, duration, buffer, seekRange });
```

File: js/google_ima.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply js/google_ima.js]: ahahah, yes, of course))

File: js/google_ima.js [hunk 3]
```diff
@@ -2894,13 +2894,29 @@ require("../html5-common/js/utils/utils.js");
...
+      const seekRange = { begin: 0, end: 0 };
+      notifyIfInControl(
+        eventname,
         { "currentTime" : currentTime,
           "duration" : duration,
-          "buffer" : 0,
-          "seekRange" : { "begin" : 0, "end" : 0 } });
+          "buffer" : buffer,
+          "seekRange" : seekRange
+        }
```
Comment (zelibobla) [inline js/google_ima.js]:
```suggestion
        notifyIfInControl(eventName, { currentTime, duration, buffer, seekRange });
```

**Pull request merged**